scripts/packages/VSCodeServer/src/misc.jl

```diff
@@ -14,7 +14,7 @@ function Base.write(io::LimitIO, v::UInt8)
     io.n += write(io.io, v)
 end
 
-function sprintlimited(args...; func = show, limit::Int = 30, ellipsis::AbstractString = "…", color = false)
+function sprintlimited(args...; func=show, limit::Int=30, ellipsis::AbstractString="…", color=false)
```

Comment:
Do we prefer not having whitespaces around arguments ?

Reply:
DocumentFormat.jl does, but I think we might just have shipped a version where we can configure that? Not sure, if we want to change, we should probably change for the whole code base in a separate PR? This code formatting was done automatically at save.